CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 0.5.1
+
+- Distinguish between example and swagger, and ignore analyzing '\$ref' in example file.
```

Comment:
Should we flag this when example file has '$ref' referenced?

Reply:
No. It's ok to have `$ref` in an example.